libpod/container_internal_linux.go

```diff
@@ -1698,7 +1711,12 @@ func (c *Container) generateResolvConf() (string, error) {
 		nameservers = resolvconf.GetNameservers(resolv.Content)
 		// slirp4netns has a built in DNS server.
 		if c.config.NetMode.IsSlirp4netns() {
-			nameservers = append([]string{slirp4netnsDNS}, nameservers...)
+			slirp4netnsDNS, err := GetSlirp4netnsDNS(c.slirp4netnsSubnet)
+			if err != nil {
+				logrus.Error("failed to determine Slirp4netns DNS: ", err.Error())
```

Comment:
Warning not Error.

Reply:
Fixed: ef6027b2db9d7ebd4a294af002c08c0c93d0e5db